docs/tutorials/sd-card-removal.md

```diff
@@ -0,0 +1,25 @@
+---
+title: SD card Removal
```

Comment:
When I first read this, it made me think it was about how to "safely remove" (via software) the SD card. Might be better to say this is about physically removing the card.